bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitCommandExtension.java

```diff
@@ -42,6 +46,7 @@
     private static final String LEGACY_SUBCMD_PRINT_ACCESSORY = "printAccessory";
 
     private final Logger logger = LoggerFactory.getLogger(HomekitCommandExtension.class);
+    @Nullable
     private Homekit homekit;
```

Comment:
We use the convention to inline the Non null annotations:
```suggestion
    private @NonNullByDefault({}) Homekit homekit;
```

Reply:
good point. i have not paid attention to this until now. 
so, made all nullable and nonNull annotation inline.